File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.java
import java.util.*;

class Solution {
    public String solution(String my_string) {
        StringBuilder sb = new StringBuilder();
        char[] arr = my_string.toCharArray();
        
        for(char a:arr){
            if(Character.isUpperCase(a))
                sb.append(String.valueOf(a).toLowerCase());
            else
                sb.append(String.valueOf(a).toUpperCase());
        }
        
        String answer = sb.toString();
        return answer;
    }
}

File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.js
function solution(my_string) {
    var _my_string = my_string.split("");
    var answer = [];
    
    for(var m of _my_string){
        if (m === m.toUpperCase()) {
            answer.push(m.toLowerCase());
        } else {
            answer.push(m.toUpperCase());
        }
    }
    
    answer = answer.join("");    
    
    return answer;
}